CMakeLists.txt

```diff
@@ -36,6 +37,23 @@ if (MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4125")
 endif()
 
+# Enable ccache
+# LLVM_CCACHE_BUILD for consistency, but this currently has no effect
```

Comment:
What effect would it have?

Reply:
Some LLVM options have cascading effects in files included through MLIR.cmake. I don't really see what this would be for ccache, but it seems better safe than sorry to avoid a mismatch.

Reply:
I don't think there would be any impact here. If anything, the option should go the other way around and the onnx-mlir version should be set to the llvm version.